### ISSUE TITLE
l1_loss

### ISSUE THREAD

**Issue by RohitPawar2406** [unifyai/ivy#16455]

**Comment by morady0213** [comment]
loss= L1Loss()

for epoch in range(num_epochs):
    for batch in data_loader:
        inputs, targets = batch
        outputs = model(inputs)
        loss = loss(outputs, targets)
        loss.backward()
        optimizer.step()
        optimizer.clear_grad()
        print(f"Epoch {epoch+1}, Loss: {loss.numpy()}")
